pkg/asset/k8s_test.go

```diff
@@ -0,0 +1,36 @@
+package asset
+
+import (
+	"testing"
+)
+
+func TestEncodeBytes(t *testing.T) {
```

Comment:
I don't think this test buys us much. Maybe just a test on "newBootstrapToken" that ensures the resulting tokens are the right length and the right character set?

Reply:
> I don't think this test buys us much. Maybe just a test on "newBootstrapToken"...

The old code would have passed that test.  This test checks the advertized base 256 to base `len(charset)` conversion, which you need to show that we're not unevenly collapsing the input distribution.